docs/content/docs/getting-started/build.md

```diff
@@ -154,11 +154,11 @@ cd LocalAI
 # build the binary
 make build
 
-# Download gpt4all-j to models/
-wget https://gpt4all.io/models/ggml-gpt4all-j.bin -O models/ggml-gpt4all-j
+# Download phi-2 to models/
+wget https://huggingface.co/TheBloke/phi-2-GGUF/resolve/main/phi-2.Q2_K.gguf -O models/phi-2.Q2_K
 
```

Comment:
updated to `phi-2` as requested in the issue, it's only half the size of `ggml-gpt4all-j`, easier to download